index.html

```diff
@@ -15,305 +15,364 @@
     <header class="top">
       <a class="active" href="#"><img src="img/Mylogo.png" alt="" /></a>
       <a href="#" class="toggleNav">☰</a>
+      <nav>
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><a href="#">About</a></li>
+        </ul>
+      </nav>
     </header>
+
     <main>
       <section class="backimg">
-        <div class="title">
-          <h1>
-            I'm Ricardo <br />
-            Glad to see <br />
-            you
-          </h1>
-        </div>
-        <div>
-          <p class="para">
-            I’m a software developer! I can help you build a product, feature or
-            website Look through some of my work and experience! If you like
-            what you see and have a project you need coded, don’t hesitate to
-            contact me.
-          </p>
-        </div>
-        <div>
-          <a href="#" class="connect">Let's connect</a>
-        </div>
-        <div class="cont">
-          <ul class="icons">
-            <li class="list1">
-              <a href="#">
-                <img src="./img/github.svg" alt="github logo" />
-              </a>
-            </li>
-            <li class="list1">
-              <a href="#">
-                <img src="./img/Linkedin.svg" alt="Linkedin logo" />
-              </a>
-            </li>
-            <li class="list1">
-              <a href="#">
-                <img src="./img/twitter.svg" alt="Twitter logo" />
-              </a>
-            </li>
-            <li class="list1">
-              <a href="#">
-                <img src="./img/medium.svg" alt="medium logo" />
-              </a>
-            </li>
-            <li class="list1">
-              <a href="#">
-                <img src="./img/look.svg" alt=" logo" />
-              </a>
-            </li>
-          </ul>
+        <div class="container">
+          <div class="title">
+            <h1>
+              I'm Ricardo <br />
+              Glad to see <br class="br" />
+              you!
+            </h1>
+          </div>
+          <div>
+            <p class="para">
+              I’m a software developer! I can help you build a product, feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
```

Comment:
Kindly apply the given design as shown below

### Original Design
![right1](https://user-images.githubusercontent.com/58376020/121755073-8bcb2f00-cb16-11eb-9bd7-232df2630659.png)

### Your project
![wrong1](https://user-images.githubusercontent.com/58376020/121755105-9d143b80-cb16-11eb-92ad-ef21a59f6fe3.png)